modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/portlet/FDSViewsPortlet.java

```diff
@@ -494,16 +531,22 @@ private void _createFDSSortObjectDefinition(
 						_language.get(locale, "sorting"), "sortingDirection",
 						true)));
 
-		_objectDefinitionLocalService.publishSystemObjectDefinition(
-			userId, fdsSortObjectDefinition.getObjectDefinitionId());
+		ObjectRelationship fdsViewFDSSortRelationship =
+			_objectRelationshipLocalService.addObjectRelationship(
+				userId, fdsViewObjectDefinition.getObjectDefinitionId(),
+				fdsSortObjectDefinition.getObjectDefinitionId(), 0,
+				ObjectRelationshipConstants.DELETION_TYPE_CASCADE,
+				LocalizedMapUtil.getLocalizedMap(
+					"FDSView FDSSort Relationship"),
+				"fdsViewFDSSortRelationship", false,
+				ObjectRelationshipConstants.TYPE_ONE_TO_MANY);
 
-		_objectRelationshipLocalService.addObjectRelationship(
-			userId, fdsViewObjectDefinition.getObjectDefinitionId(),
-			fdsSortObjectDefinition.getObjectDefinitionId(), 0,
-			ObjectRelationshipConstants.DELETION_TYPE_CASCADE,
-			LocalizedMapUtil.getLocalizedMap("FDSView FDSSort Relationship"),
-			"fdsViewFDSSortRelationship", false,
-			ObjectRelationshipConstants.TYPE_ONE_TO_MANY);
+		long[] relationshipId = new long[1];
+
+		relationshipId[0] =
+			fdsViewFDSSortRelationship.getObjectRelationshipId();
+
+		_objectDefinitionLocalService.bindObjectDefinitions(relationshipId);
```

Comment:
As seen in https://github.com/liferay-objects/liferay-portal/pull/2458, `bindObjectDefinitions` needs a list of edges (object relationship ids) where the last item is the root, the penult is the edge depth 1, antepenult is the edge depth 2 and so on.